Automattic/wp-calypso
# Translate the product name in the list of features on /plans page

## Proposed Changes

Pass the locale to the v1.5/plans API route to retrieve translated values for the `product_name_short` field.
Fixes an issue with the /plans page where the plan name in "Everything in {planName}, plus" was appearing untranslated.

## Testing Instructions

1. Checkout the PR locally or use the calypso.live instance.
2. Switch to a Mag-16 locale where the Free plan name is translated, like Spanish or German.
3. Go to /plans/<your site> and confirm that the Free plan name is translated.
4. Repeat step 3 on wordpress.com to check the original issue.

Before:
<img width="763" alt="Screenshot 2024-01-04 at 00 02 04" src="https://github.com/Automattic/wp-calypso/assets/7847633/59a32479-6281-48de-9aaa-53bf4ce7bb15">

After:
<img width="763" alt="Screenshot 2024-01-04 at 00 03 21" src="https://github.com/Automattic/wp-calypso/assets/7847633/31026ea8-0fc8-4754-9dc3-ea58ffbc2c94">




*

## Pre-merge Checklist
